README.md

```diff
@@ -14,3 +14,15 @@ git checkout develop
 git merge upstream/develop
 git push
 ~~~~
+
+## How to update your fork when the code you forked from changes
+Here is how you update the develop branch of your fork to get the latest changes from the main spectre repo (called the "upstream repo."
+~~~~
+# Set up upstream (first time only)
+git remote add upstream ORIGINAL-REPO-LOCATOR
+# Update a fork
+git fetch upstream
+git checkout develop
+git merge upstream/develop
+git push
```

Comment:
Why did you do this?

Reply:
If you don't push, your squash is only on your local checkout, not shared with GitHub.